test/test_imagestreams_quickstart.py

```diff
@@ -31,18 +31,20 @@
 
 TAGS = {
     "rhel8": "-ubi8",
-    "rhel9": "-ubi9"
+    "rhel9": "-ubi9",
```

Comment:
Can we share this dictionary from a single source file?

Reply:
Sure. I will do it.

Reply:
Yes, de-duplicate it somewhere.

Reply:
Fixed by https://github.com/sclorg/s2i-python-container/pull/734/commits/2d4a4f8d34010f7ca89b9a8e236906d0f59c9369